rmf_door_msgs/msg/DoorRequest.msg

```diff
@@ -0,0 +1,7 @@
+builtin_interfaces/Time request_time
+string requester_id
+string door_name
```

Comment:
Should we not use `DoorState desired_state` here instead of repeating the same fields?

Reply:
Oh I guess the `DoorState` has a time field :thinking:

Reply:
Yeah, there are fair arguments either way. I'm just afraid of the enums of the door mode drifting because of multiple sources of truth.